x-pack/plugins/reporting/server/browsers/chromium/paths.ts

```diff
@@ -7,32 +7,40 @@
 import path from 'path';
 
 export const paths = {
-  archivesPath: path.resolve(__dirname, '../../../.chromium'),
+  archivesPath: path.resolve(__dirname, '../../../../../../.chromium'),
   baseUrl: 'https://storage.googleapis.com/headless_shell/',
   packages: [
     {
       platforms: ['darwin', 'freebsd', 'openbsd'],
-      archiveFilename: 'chromium-312d84c-darwin.zip',
+      architecture: 'x64',
+      archiveFilename: 'chromium-312d84c-darwin_x64.zip',
       archiveChecksum: '020303e829745fd332ae9b39442ce570',
       binaryChecksum: '5cdec11d45a0eddf782bed9b9f10319f',
       binaryRelativePath: 'headless_shell-darwin/headless_shell',
     },
     {
       platforms: ['linux'],
-      archiveFilename: 'chromium-312d84c-linux.zip',
+      architecture: 'x64',
+      archiveFilename: 'chromium-312d84c-linux_x64.zip',
       archiveChecksum: '15ba9166a42f93ee92e42217b737018d',
       binaryChecksum: 'c7fe36ed3e86a6dd23323be0a4e8c0fd',
       binaryRelativePath: 'headless_shell-linux/headless_shell',
     },
+    {
+      platforms: ['linux'],
+      architecture: 'arm64',
+      archiveFilename: 'chromium-312d84c-linux_arm64.zip',
+      archiveChecksum: 'a8456468f4da204985c947d2e5c4b669',
+      binaryChecksum: '7fdccff319396f0aee7f269dd85fe6fc',
+      binaryRelativePath: 'headless_shell-linux/_/arm64/headless_shell',
```

Comment:
This path seems odd, is there a reason that there's a blank `_` folder?

Reply:
Upon further investigation, this was my bad and I have fixed.